api/src/main/resources/workbench-api.yaml

```diff
@@ -4610,6 +4610,16 @@ definitions:
         description: 'All the selected domain/value pairs in the data set'
         items:
           "$ref": "#/definitions/DomainValuePair"
+  DataSetGenerateCodeRequest:
```

Comment:
I think the easiest way to fix this would be create a new endpoint that receives this as a request with the `DataSet.id`. The new endpoint will call a service method that reads the dataset by primary key(id) and calls the proper version of `DataSetServiceImpl#domainToBigQueryConfig`